nixos/doc/manual/from_md/installation/installing-usb.section.xml

```diff
@@ -1,35 +1,82 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="sec-booting-from-usb">
-  <title>Booting from a USB Drive</title>
+  <title>Booting from a USB flash drive</title>
   <para>
-    For systems without CD drive, the NixOS live CD can be booted from a
-    USB stick. You can use the <literal>dd</literal> utility to write
-    the image: <literal>dd if=path-to-image of=/dev/sdX</literal>. Be
-    careful about specifying the correct drive; you can use the
-    <literal>lsblk</literal> command to get a list of block devices.
+    The image has to be written verbatim to the USB flash drive for it
+    to be bootable on UEFI and BIOS systems. Here are the recommended
+    tools to do that.
   </para>
-  <note>
-    <title>On macOS</title>
+  <section xml:id="sec-booting-from-usb-graphical">
+    <title>Creating bootable USB flash drive with a graphical
+    tool</title>
+    <para>
+      Etcher is a popular and user-friendly tool. It works on Linux,
+      Windows and macOS.
+    </para>
+    <para>
+      Download it from
+      <link xlink:href="https://www.balena.io/etcher/">balena.io</link>,
+      start the program, select the downloaded NixOS ISO, then select
+      the USB flash drive and flash it.
+    </para>
+    <warning>
+      <para>
+        Etcher reports errors and usage statistics by default, which can
+        be disabled in the settings. Some users might not like that.
+      </para>
+    </warning>
+    <para>
+      An alternative is
+      <link xlink:href="https://gitlab.com/bztsrc/usbimager/#usbimager">USBImager</link>,
+      which is very simple and does not connect to the internet.
+      Download the version with write-only (wo) interface for your
+      system. Start the program, select the image, select the USB flash
+      drive and click <quote>Write</quote>.
+    </para>
+  </section>
+  <section xml:id="sec-booting-from-usb-linux">
+    <title>Creating bootable USB flash drive from a Terminal on
+    Linux</title>
+    <para>
+      First, make sure all partitions on the device are properly
+      unmounted. Then use the <literal>dd</literal> utility to write the
+      image to the USB flash drive:
+    </para>
     <programlisting>
-$ diskutil list
-[..]
-/dev/diskN (external, physical):
-   #:                       TYPE NAME                    SIZE       IDENTIFIER
-[..]
-$ diskutil unmountDisk diskN
-Unmount of all volumes on diskN was successful
-$ sudo dd if=nix.iso of=/dev/rdiskN bs=1M
+sudo umount /dev/sdX*
+sudo dd if=&lt;path-to-image&gt; of=/dev/sdX bs=4M
```

Comment:
It would be good to add the progress `status=progress`

https://wiki.archlinux.org/title/USB_flash_installation_medium#In_GNU/Linux

A lot of guides just suggest to use `cat` these days but there's some nuance to using `sudo` + `>` so I think `dd` is more straightforward

Reply:
We  discussed the command before: https://github.com/NixOS/nixpkgs/pull/175128#discussion_r884184464

@samueldr what do you think about `progress`? would it be safe to add

Reply:
```suggestion
sudo dd if=&lt;path-to-image&gt; of=/dev/sdX bs=4M status=progress
```

Reply:
It's likely fine to add, yes.

As I tried to explain in the initial (now orphaned) comment, there is a lot of cargo-culting, and selecting options is *hard*. Especially when there are multiple divergent `dd` implementations.

Though it's unlikely to cause issues to add those. Hopefully users using `dd` know what to do when commonly used options don't work.